### PR TITLE
Adding convince my boss blogpost -courtesy Shalini

### PR DESCRIPTION
**Pull request by sharmi**
https://github.com/pythonindia/inpycon-blog/issues/213

### CODE REVIEW

File: content/blog/convince-my-boss.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Title: Convince Your Boss With This Email : PyCon India 2019 Date: 2019-06-01 09:00 Category: 2019 Slug: convince-your-boss-with-this-email-pycon-india-2019 Summary: Save time composing a email to convince your boss to send you to PyCon India. We got you covered.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: This part needs reformatting.

File: content/blog/convince-my-boss.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+There a gazillion things to be gained attending PyCon India. Unfortunately, it takes time to sit and compose a neat little email to send to your boss enumerating them. Now consider that atleast half of our attendees attempt to do that, we would lose so many hours of productivity reinventing the wheel ;) . So PyCon India is here to save the day. We have composed a convince-the-boss letter to save you time. Of course, feel free to edit it as you like, remove some, add some or use as is. 
+
+## Important notes:
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Use title case: Important Notes

File: content/blog/convince-my-boss.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+1. You do not need to keep all the points. Remove ones you deem unnecessay. The shorter it is, the better the chances that your boss will read it.
+
+2. Watchout for the {TEMPLATE_VARIABLES}. You need to replace it with information relevant to you. Don't let one escape you.
+
+## The letter
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Use title case: The Letter

File: content/blog/convince-my-boss.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Dear {BOSSNAME},
+
+I am excited to brief you about PyCon India, India's premier conference on using and developing the Python programming language that takes place in Chennai from October 12-15, 2019. About a 1000+ of the industry's Pythonistas from different fields are bound to be in attendance for the 11th edition of this mega event.
+
+It would be great if I get an opportunity to attend this conference as there will be innumerable benefits for me as an individual which will in turn impact the organization as well. To list a few:
+
+* (Optional) Am Particularly keen on attending this {TALK/WORKSHOP/DEVSPRINT} on {TOPIC} presented by {SPEAKER-NAME} as our project is based on this {OR UPCOMING PROJECT BASED ON THIS}.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Particularly -> particularly

File: content/blog/convince-my-boss.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* Exposure to the actionable best practices & tools that we could use in our projects, saving money and time spent exploring different options.
+* Provides a forum to get a sneak peek at upcoming innovations.
+* Provides a platform to unmask the rationality behind the trending technologies like ML/AI, Automation, Web development, Software development, IOT and more that uses Python.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: that use Python.

File: content/blog/convince-my-boss.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+* Learning Technologies from a wide array of Talks, Workshops, Keynotes speech, Developer Sprints each presented by inspiring and enthusiastic Industry experts, community leaders and many young minds delving into new technologies.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Unwanted capitalization:
  * Learning Technologies -> Learning technologies
  * Industry experts -> industry experts

File: content/blog/convince-my-boss.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+* Opportunity to network with like-minded people from diverse backgrounds that includes Geek Programmers, Enthusiasts, Entrepreneurs, Scholars, Intellectual and many more.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Geek Programmers -> geek programmers
Ethusiasts -> enthusiasts
Entrepreneurs -> entrepreneurs
Scholars -> scholars
Intellectual -> intellectuals

File: content/blog/convince-my-boss.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+* A chance to spread the word about our Company and create goodwill about the opportunities and incentives the employees are given. This will be the ideal way to attract the future employees from an diversified talent pool.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Company -> company
the future employees -> future employees

File: content/blog/convince-my-boss.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+* {OPTIONAL - } As a _______{WOMAN/, attending the conference will also demonstrate to the tech community that our company takes diversity in our workforce very seriously. Innovative thinking and better products come from teams that are diverse covering more ground.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Did we miss something here? Is there an option apart from the WOMAN?

File: content/blog/convince-my-boss.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+There is an Early Bird tickets offer on for the first 300 tickets sold before 30th June 2019 and it costs only Rs 1100. Booking later (or after 300 tickets)would cost Rs 1600. The tickets include access to all PyCon India keynotes & talks, sponsor showcase, community spaces {if included this time}, an official PyCon India T-shirt along with lunch and high-tea in the evening for both days (12th-13th October).
+
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Need to add a space after the close brace.
community spaces {if included this time} -> open spaces

File: content/blog/convince-my-boss.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+I would be happy to write a post-conference report detailing what I've learnt and to recommend any changes that we could implement to improve our practices. Please let me know if you have any questions or concerns. I hope we make use this opportunity to benefit our company.
```
Comment (bravegnu) [inline content/blog/convince-my-boss.md]: Grammar fixes? 'I hope we can make use of this opportunity to benefit our company.'

File: content/blog/convince-my-boss.md (hunk 1 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: This is as it was in the other blog posts. I just copied it in. Should I add newlines or is something else missing?

File: content/blog/convince-my-boss.md (hunk 2 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: Done

File: content/blog/convince-my-boss.md (hunk 3 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: Done

File: content/blog/convince-my-boss.md (hunk 4 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: Done

File: content/blog/convince-my-boss.md (hunk 5 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: Done

File: content/blog/convince-my-boss.md (hunk 7 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: Done

File: content/blog/convince-my-boss.md (hunk 6 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: Done

File: content/blog/convince-my-boss.md (hunk 1 above)
Comment (sharmi) [reply content/blog/convince-my-boss.md]: Ok I see it now Vijay.

**Comment by bravegnu** [comment]
Superseded by #215

**Pull request closed without merging**